.github/workflows/odoo-ci-cd.yml

```
@@ -39,3 +39,35 @@
+    steps:
+      - name: List GHCR tags for a container image
+
+        env:
+          GHCR_OWNER: my-org               # <- Change this
+          IMAGE_NAME: my-app               # <- Change this
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+              # GitHub token
+              TOKEN=$GH_TOKEN
+              IMAGE_PREFIX="ghcr.io/gldzzpro/odoo17"
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)